v0/destinations/mailmodo/transform.js

```diff
@@ -0,0 +1,297 @@
+const get = require("get-value");
+const { EventType } = require("../../../constants");
+const {
+  ConfigCategory,
+  IDENTIFY_MAX_BATCH_SIZE,
+  mappingConfig,
+  BASE_URL,
+  DESTINATION
+} = require("./config");
+const {
+  defaultRequestConfig,
+  constructPayload,
+  isDefinedAndNotNullAndNotEmpty,
+  defaultPostRequestConfig,
+  defaultBatchRequestConfig,
+  removeUndefinedAndNullValues,
+  CustomError,
+  getErrorRespEvents,
+  getSuccessRespEvents,
+  generateErrorObject
+} = require("../../util");
+const { TRANSFORMER_METRIC } = require("../../util/constant");
+const { deduceAddressFields } = require("./utils");
+
+const responseBuilder = responseConfgs => {
+  const { resp, apiKey, endpoint } = responseConfgs;
+
+  const response = defaultRequestConfig();
+  if (resp) {
+    const responseBody = "JSON";
+    response.endpoint = `${BASE_URL}${endpoint}`;
+    response.headers = {
+      mmApiKey: `${apiKey}`,
+      "Content-Type": "application/json"
+    };
+    response.method = defaultPostRequestConfig.requestMethod;
+    response.body[`${responseBody}`] = removeUndefinedAndNullValues(resp);
+  }
+  return response;
+};
+
+const identifyResponseBuilder = (message, { Config }) => {
+  const { apiKey } = Config;
+  let { listName } = Config;
+
+  const payload = constructPayload(
+    message,
+    mappingConfig[ConfigCategory.IDENTIFY.name]
+  );
+
+  if (!payload.email) {
+    throw new CustomError("Email is required for identify call", 400);
+  }
```

Comment:
No need to check it here you can make it required: true in identify config.json

Reply:
Removed